charts/k8s-ovpn-chart/templates/deployment.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "k8s-ovpn-chart.fullname" . }}
-  namespace: {{ .Values.namespace.name }}
```

Comment:
What's the reason for removing `namespace` here?

Reply:
updated